pkg/config/sample-config.txt

```diff
@@ -0,0 +1,26 @@
+version = 31
+credential.accessKey = xxx
+credential.secretKey = yyy    ## SecretKey cannot be changed
+region = us-east-1
+browser = on
```

Comment:
We don't honor browser in our config anymore we can remove this entry..

Reply:
OK